src/Sarif.Viewer.VisualStudio.Core/Themes/InformationStyles.xaml

```diff
@@ -16,7 +16,7 @@
     </Style>
 
     <Style x:Key="RuleTitlePanelStyle" TargetType="DockPanel">
-        <Setter Property="Margin" Value="0,0,0,5"/>
+        <Setter Property="Margin" Value="0,0,0,10"/>
```

Comment:
>10 [](http://example.com/codeflow?start=47&length=2)

all these UI changes, you have run and see them align right

Reply:
yes, there is a screenshot of the sarif explorer in the PR page pls review.